widget/view.controller.js

```diff
@@ -68,19 +77,55 @@
             scheduleFrequency: null
         };
         $scope.outbreakAlertSeverityList = ['Critical', 'High', 'Medium'];
-        $scope.$watch('activeTab', function ($newTab, $oldTab) {
-            if (!$oldTab) {
-                // skip first run
-                return;
-            }
-            $state.go('.', {
-                tab: $scope.selectedEnv.huntTools[0].itemValue
-            }, {
-                notify: false,
-                location: 'replace'
+
+        $scope.parent_wf_id = '';
+        var subscription;
+        initWebsocket();
+
+        $scope.$on('websocket:reconnect', function () {
+            initWebsocket();
+        });
+
+        function initWebsocket() {
+            websocketService.subscribe('runningworkflow', function (data) {
+                if (data.parent_wf === 'null' && $scope.isPlaybookExecuted === false) {
```

Comment:
null value should not be as string. Please check once

Reply:
verified in UI code